File: src/sdk-reference/cpp/1/realtime/unsubscribe/snippets/unsubscribe.cpp
kuzzleio::NotificationListener listener =
  [](const std::shared_ptr<kuzzleio::notification_result> &notification){};

try {
  std::string room_id = kuzzle->realtime->subscribe(
    "nyc-open-data",
    "yellow-taxi",
    "{}",
    &listener);

  kuzzle->realtime->unsubscribe(room_id);

  std::cout << "Successfully unsubscribed" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/realtime/unsubscribe/snippets/unsubscribe.cs
NotificationListener listener =
  [](const notification_result *notification){};

try {
  string room_id = kuzzle.realtime.subscribe(
    "nyc-open-data",
    "yellow-taxi",
    "{}",
    listener);

  kuzzle.realtime.unsubscribe(room_id);

  Console.WriteLine("Successfully unsubscribed");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}
